lms/djangoapps/courseware/views/views.py

```diff
@@ -1685,9 +1682,7 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
             'staff_access': bool(request.user.has_perm(VIEW_XQA_INTERFACE, course)),
             'xqa_server': settings.FEATURES.get('XQA_SERVER', 'http://your_xqa_server.com'),
             'display_reset_dates_banner': display_reset_dates_banner,
-            'reset_deadlines_url': reset_deadlines_url,
-            'reset_deadlines_redirect_url_base': reset_deadlines_redirect_url_base,
-            'reset_deadlines_redirect_url_id_dict': {'course_id': str(course.id), 'usage_key_string': usage_key_string}
```

Comment:
Don't we still need to pass these in? For use in the `reset_deadlines_banner.html` file

Reply:
no because these are the attributes for the button.  we are now only showing a link that will redirect the user to the web app.